components/CaseStudiesProject.tsx

```diff
@@ -1,37 +1,59 @@
+"use client";
```

Comment:
I don't believe you need "use client" here.

Reply:
Because of Framer motion, I need that.